bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitTaggedItem.java

```diff
@@ -145,6 +161,78 @@ public HomekitOHItemProxy getProxyItem() {
         return proxyItem;
     }
 
+    /**
+     * Send DecimalType command to a NumberItem (or a Group:Number)
+     * 
+     * @param command
+     */
+    public void send(DecimalType command) {
+        if (getItem() instanceof GroupItem && getBaseItem() instanceof NumberItem) {
+            ((GroupItem) getItem()).send(command);
+            return;
+        } else if (getItem() instanceof NumberItem) {
+            ((NumberItem) getItem()).send(command);
+            return;
+        }
+        logger.warn("Received DecimalType command for item {} that doesn't support it. This is probably a bug.",
+                getName());
+    }
+
+    /**
+     * Send OnOffType command to a SwitchItem (or a Group:Switch)
+     * 
+     * @param command
+     */
+    public void send(OnOffType command) {
+        if (getItem() instanceof GroupItem && getBaseItem() instanceof SwitchItem) {
+            ((GroupItem) getItem()).send(command);
+            return;
+        } else if (getItem() instanceof SwitchItem) {
+            ((SwitchItem) getItem()).send(command);
+            return;
+        }
+        logger.warn("Received OnOffType command for item {} that doesn't support it. This is probably a bug.",
+                getName());
+    }
+
+    /**
+     * Send PercentType command to a DimmerItem or RollershutterItem (or a Group:Switch/Group:Rollershutter)
```

Comment:
Is it really Group:Switch ?
Maybe more Group:Dimmer ?

Reply:
Absolutely right. Minor brain fart when going through all of these. PR updated.